pkg/agent/cniserver/pod_configuration.go

```diff
@@ -390,49 +204,30 @@ func (pc *podConfigurator) configureInterfaces(
 		}
 	}()
 
-	// Note that configuring IP will send gratuitous ARP, it must be executed
-	// after Pod Openflow entries are installed, otherwise gratuitous ARP would
-	// be dropped.
-	klog.V(2).Infof("Configuring IP address for container %s", containerID)
-	if err = configureContainerAddr(netns, containerIface, result); err != nil {
-		return fmt.Errorf("failed to configure IP address for container %s: %v", containerID, err)
-	}
-
 	containerConfig.OVSPortConfig = &interfacestore.OVSPortConfig{PortUUID: portUUID, OFPort: ofPort}
 	// Add containerConfig into local cache
 	pc.ifaceStore.AddInterface(containerConfig)
+
+	// Note that the gratuitous ARP must be executed after Pod Openflow entries are installed, otherwise gratuitous
```

Comment:
`the gratuitous ARP must be executed` is not a correct grammar, and you should not say GARP any more as it's specific to linux?